eng/scripts/Language-Settings.ps1

```diff
@@ -67,7 +67,8 @@ function Get-AllPackageInfoFromRepo ([string] $ServiceDirectory) {
   }
   catch {
     # This is soft error and failure is expected for python metapackages
-    LogError "Failed to get all package properties"
+    LogError "Failed to get all package properties. $($_.Exception.Message)"
+    return $null
```

Comment:
1. Nit: 
```suggestion
    LogError "Failed to get all package properties: $($_.Exception.Message)"
    return $null
```
2. `return $null`? Seems rather exceptional - terminal, even. Or is the caller checking for valid data, which begs the question why not just throw here?